.rubocop_todo.yml

```diff
@@ -326,32 +347,57 @@ Style/LeadingCommentSpace:
 Style/LineEndConcatenation:
   Enabled: false
 
-# Offense count: 32
+# Offense count: 43
 # Cop supports --auto-correct.
 Style/MethodCallParentheses:
   Enabled: false
 
-# Offense count: 7
+# Offense count: 8
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, SupportedStyles.
 # SupportedStyles: require_parentheses, require_no_parentheses, require_no_parentheses_except_multiline
 Style/MethodDefParentheses:
   Enabled: false
 
-# Offense count: 45
+# Offense count: 5
+Style/MethodMissing:
+  Enabled: false
```

Comment:
I'm going to have to leave it disabled due to https://github.com/bbatsov/rubocop/issues/3358 but I fixed everything so this should be enabled on the next rubocop upgrade.